test/CDB_JenksBinsTest_expect

```diff
@@ -1,10 +1,18 @@
-1
-5
-10
-20
-20
-1
-5
-10
-20
-
+1.01
+5.01
+10.01
+15.01
+20.1
+1.01
+5.01
+10.01
+15.01
+20.1
+0.99
+4.99
+10.01
+14.99
+19.9
+0.99
+1.0
+10.01
```

Comment:
Why are there only three values here when the test asks for 4 bins?
```sql
WITH data_small AS (
    SELECT Array[0.99, 1.0, 10.01, 10.01, 10.01, 10.01]::numeric[] AS s
)
SELECT unnest(CDB_JenksBins(s, 4)) FROM data_small;
```

Reply:
There are only 3 different values (0.99, 1.0 and 10.01) so it's impossible to return 4 different bins.